starter_code/seed.js

```diff
@@ -1,13 +1,79 @@
 import mongoose from "mongoose";
 import "./db-connect.js";
-import { Customer } from "./models.js";
-
+import Pizza from './models/Pizza.js'
+import Customer from "./models/Customer.js";
+import Order from "./models/Order.js";
+import faker from 'faker'
 
 (async () => {    
     
   try {
     // seed some customers...
+    await Pizza.deleteMany()
+    await Customer.deleteMany()
+    await Order.deleteMany()
+
+    const pizzaPromises = Array(5)
+      .fill(null)
+      .map(() => {
+        const pizza = {
+          name: `${faker.random.word()} ${faker.address.state()}`,
+          price: faker.commerce.price(7.50, 15.90)
```

Comment:
Nicely used faker for some reasonable price range.

But for 15.90 (!) I expect really greatest Pizza in Berlin :)